src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
 // export const Person = ({ person }) => ();
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age && <p className="Person__age">I am {person.age} years old</p>}
```

Comment:
Remove it for fix failed test 
<img width="567" height="337" alt="image" src="https://github.com/user-attachments/assets/724765b1-245a-410e-877e-f7cb443995ad" />

```suggestion
    {person.age && <p className="Person__age">I am {person.age}</p>}
```